src/qibo_client/qibo_client.py

```diff
@@ -142,11 +142,13 @@ def check_client_server_qibo_versions(self):
         qibo_server_version = response.json()["qibo_version"]
         qibo_local_version = qibo.__version__
 
-        msg = (
-            "Local Qibo package version does not match the server one, please "
-            f"upgrade: {qibo_local_version} -> {qibo_server_version}"
-        )
-        assert qibo_local_version == qibo_server_version, msg
+        if qibo_local_version != qibo_server_version:
```

Comment:
As discussed we should place a minimum version, e.g. something like:
```suggestion
        if qibo_local_version >= qibo_server_version:
```

Reply:
I'd better put `qibo_local_version <= qibo_server_version`, then raise the warning.

Reply:
I think we should force local versions to be equal or larger than server, the latter when we are developing the software or releasing to pypi. If the user is using a lower version the outcome is unknown, so we should not allow executions with `local <= server`.

Reply:
Ok, the if condition should warn if `local != server` and raise error if `local < server`. Right?

Reply:
True, I was forgetting. We put `0.2.4` as the minimum version and then raise an error if there's a mismatch

Reply:
Yes, exactly.